boards/arm/nrf52840_pca10056/nrf52840_pca10056.dts

```diff
@@ -74,6 +74,10 @@
 	};
 };
 
+arduino_i2c: &i2c0 {};
+arduino_spi: &spi1 {};
+arduino_serial: &uart0 {};
```

Comment:
This is not going to work just like that. The pins assigned to these peripherals must also match the Arduino pinout. And currently they do only for I2C.
For the nrf52840_pca10056 board the SPI pins would need to be assigned as follows:
- SCK (Arduino digital pin 13): 47 (p1.15)
- MOSI (D12): 46 (p1.14)
- MISO (D11): 45 (p1.13)
- SS (D10): 44 (p1.12)

and the UART pins:
- TXD (D1): 32 (p1.02)
- RXD (D0): 31 (p1.01)

Otherwise, there is no point in creating these aliases, IMO.